### DISCUSSION TITLE
Fail to create a new post.

### DISCUSSION THREAD

**Discussion by cekxm** [alshedivat/al-folio#2859] [Q&A]
I followed the instructions in [INSTALL.md](https://github.com/cekxm/cekxm.github.io/blob/main/INSTALL.md) and set up the github page successfully. 
Here are what I did to create a new post which failed.

1. clone the repository to my computer
2. create a new markdown file in the _posts folder. Its name is 2024-11-26-code2.md
```
---
layout: post
title: a post with code
date: 2024-11-26 12:13:00
---

This theme implements a built-in Jekyll feature, the use of Rouge, for syntax highlighting.
```
4. commit the change and push to github
5. wait many minutes

Nothing happen. No new post at all.
What did I miss?

**Comment by yayalingo** [comment]
Same here, i created a new one and success, created second, not shown in posts page

**Comment by cekxm** [comment]
It turns out that the post is counted as a future post because the timezone of github page is different from mine.

I will close the question.
